js/adapt-contrib-trickle.js

```diff
@@ -81,34 +79,32 @@ define([
 
                 var scrollToId = "";
                 switch (scrollTo.substr(0,1)) {
-                case "@":
-                    //NAVIGATE BY RELATIVE TYPE
-                    
-                    //Allows trickle to scroll to a sibling / cousin component relative to the current trickle item
-                    var relativeModel = fromModel.findRelative(scrollTo, {
-                        filterNotAvailable: true
-                    });
-                    
-                    if (relativeModel === undefined) return;
-                    scrollToId = relativeModel.get("_id");
-
-                    //console.log("trickle scrolling to", scrollToId, "from", fromModel.get("_id"));
-
-                    break;
-                case ".":
-                    //NAVIGATE BY CLASS
-                    scrollToId = scrollTo.substr(1, scrollTo.length-1);
-                    break;
-                default: 
-                    scrollToId = scrollTo;
+                    case "@":
+                        // NAVIGATE BY RELATIVE TYPE
+
+                        // Allows trickle to scroll to a sibling / cousin component
+                        // relative to the current trickle item
+                        var relativeModel = fromModel.findRelativeModel(scrollTo, {
+                            filterNotAvailable: true
```

Comment:
I don't see `filterNotAvailable` as an implemented option in `AdaptModel.findRelativeModel`... there's `options.filter` which takes a function..?

Reply:
I say...